Cargo.toml

```diff
@@ -32,7 +35,7 @@ usvg = { path = "usvg", version = "0.23.0", default-features = false }
 once_cell = "1.5"
 
 [features]
-default = ["filter", "text", "system-fonts", "memmap-fonts"]
+default = ["filter", "text", "system-fonts", "memmap-fonts", "gif", "jpeg-decoder", "png"]
```

Comment:
I don't think we need that level of control. The `image` feature would be enough.